target/linux/ramips/dts/mt7621_plasmacloud_pax1800-lite.dts

```diff
@@ -0,0 +1,2219 @@
+							/bits/ 8 <1 38 38 60 60 38 60 60 60 60 60>,
+							/bits/ 8 <1 60 60 60 60 60 60 60 60 60 60>;
+						paths-ru-bf =
+							/bits/ 8 <4 38 38 60 60 38 60 60 60 60 60>,
+							/bits/ 8 <1 40 40 60 60 40 60 60 60 60 60>,
+							/bits/ 8 <1 38 38 60 60 38 60 60 60 60 60>,
+							/bits/ 8 <1 60 60 60 60 60 60 60 60 60 60>;
+					};
+					b11 {
+						channels = <165 165>;
+						rates-cck =
+							/bits/ 8 <60 60 60 60>;
+						rates-ofdm =
+							/bits/ 8 <39 39 39 39 39 39 39 39>;
+						rates-mcs =
+							/bits/ 8 <1 38 38 38 38 38 38 38 38 38 38>,
+							/bits/ 8 <3 60 60 60 60 60 60 60 60 60 60>;
+						rates-ru =
+							/bits/ 8 <4 38 38 38 38 38 38 38 38 38 38 38 38>,
+							/bits/ 8 <3 60 60 60 60 60 60 60 60 60 60 60 60>;
+						paths-cck =
+							/bits/ 8 <60 60 60 60>;
+						paths-ofdm =
+							/bits/ 8 <39 39 60 60>;
+						paths-ofdm-bf =
+							/bits/ 8 <39 39 60 60>;
+						paths-ru =
+							/bits/ 8 <4 38 38 60 60 38 60 60 60 60 60>,
+							/bits/ 8 <3 60 60 60 60 60 60 60 60 60 60>;
+						paths-ru-bf =
+							/bits/ 8 <4 38 38 60 60 38 60 60 60 60 60>,
+							/bits/ 8 <3 60 60 60 60 60 60 60 60 60 60>;
+					};
+					b12 {
+						channels = <169 169>;
+						rates-cck =
+							/bits/ 8 <60 60 60 60>;
+						rates-ofdm =
+							/bits/ 8 <39 39 39 39 39 39 39 39>;
+						rates-mcs =
+							/bits/ 8 <1 38 38 38 38 38 38 38 38 38 38>,
+							/bits/ 8 <3 60 60 60 60 60 60 60 60 60 60>;
+						rates-ru =
+							/bits/ 8 <4 38 38 38 38 38 38 38 38 38 38 38 38>,
+							/bits/ 8 <3 60 60 60 60 60 60 60 60 60 60 60 60>;
+						paths-cck =
+							/bits/ 8 <60 60 60 60>;
+						paths-ofdm =
+							/bits/ 8 <60 60 60 60>;
+						paths-ofdm-bf =
+							/bits/ 8 <60 60 60 60>;
+						paths-ru =
+							/bits/ 8 <7 60 60 60 60 60 60 60 60 60 60>;
+						paths-ru-bf =
+							/bits/ 8 <7 60 60 60 60 60 60 60 60 60 60>;
+					};
+					b13 {
+						channels = <173 181>;
+						rates-cck =
+							/bits/ 8 <60 60 60 60>;
+						rates-ofdm =
+							/bits/ 8 <60 60 60 60 60 60 60 60>;
+						rates-mcs =
+							/bits/ 8 <4 60 60 60 60 60 60 60 60 60 60>;
+						rates-ru =
+							/bits/ 8 <7 60 60 60 60 60 60 60 60 60 60 60 60>;
+						paths-cck =
+							/bits/ 8 <60 60 60 60>;
+						paths-ofdm =
+							/bits/ 8 <60 60 60 60>;
+						paths-ofdm-bf =
+							/bits/ 8 <60 60 60 60>;
+						paths-ru =
+							/bits/ 8 <7 60 60 60 60 60 60 60 60 60 60>;
+						paths-ru-bf =
+							/bits/ 8 <7 60 60 60 60 60 60 60 60 60 60>;
+					};
+				};
+			};
+		};
+	};
+};
+
+&xhci {
+	status = "okay";
+};
+
+&ethernet {
+	pinctrl-0 = <&mdio_pins>;
+};
+
+&gmac0 {
+	nvmem-cells = <&macaddr_config_3fff4 0>;
+	nvmem-cell-names = "mac-address";
+	label = "dsa";
+};
+
+&switch0 {
+	ports {
+		port@4 {
+			status = "okay";
+			label = "lan";
+		};
+	};
+};
+
+&state_default {
+	gpio {
+		/* for LEDs + Button + watchdog */
+		groups = "uart3", "jtag", "wdt", "rgmii2";
+		function = "gpio";
+	};
+};
+
+&factory {
```

Comment:
this is referencing a node in the same dts file, might as well add it into the factory definition above. (preferred by committers afaik)